src/main/java/com/aws/greengrass/secretmanager/SecretManager.java

```diff
@@ -210,7 +210,7 @@ private void refreshSecretFromCloud(String arn, String versionStage) {
         boolean isSecretLabelConfigured = configurations.stream().anyMatch(
                 (secret) -> secret.getArn().contains(arn) && secret.getLabels().contains(versionStage));
```

Comment:
Should this and the other usages of versionStage in this function be versionLabel instead?

Reply:
I don't think so. I think the version label usage should be validated but I plan to include that as a future PR.